docs/admin/memory.rst

```diff
@@ -24,29 +24,28 @@ turned on by default.
 
 .. _memory-scopes:
 
-Translation memory scopes
+Translation-memory scopes
```

Comment:
Nobody calls it this way.

https://en.wikipedia.org/wiki/Translation_memory, https://omegat.org/, https://www.trados.com/solutions/translation-memory/